R/MsBackendTimsTof.R

```diff
@@ -117,30 +133,22 @@ setMethod("length", "MsBackendTimsTof", function(x) {
 
 #' @rdname MsBackendTimsTof
 setMethod("peaksData", "MsBackendTimsTof", function(object) {
```

Comment:
Isn't there an additional column that could be extracted for TimsTOF data? such as the ion mobility values? Maybe we should add here an additional parameter `columns = c("mz", "intensity")` to `peaksData` that would allow also selecting one (or more?) of these additional columns?

Reply:
Yes, there is also the "tof" variable. There are different values for this variable for a given scan. Additionaly, there is the “inv_ion_mobility" column. In this case I think that there is a unique value for each scan (although in the table returned by `opentimsr::query` is repeated as many times as the number of peaks for a scan). Should I extract this variable like m/z and intensity even though "it could be a numeric instead of a NumericList"? Should I also add separate getters for these variables like `mz` and ìntensity` for m/z and intensity?

Reply:
no additional getters and setters for these. I would just (as also described above) allow extraction of additional fields. And the `peaksData` is expected to ALWAYS return a `numeric` `matrix` with nrow being the number of peaks in a spectrum. So no conversion to `numeric` or `unique` calls or similar. Simply return what you get from `query`, split it like you do if it was `"mz"` and `"intensity"` values and return it.